.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,44 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, hotfix/**/*, release/**/* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+#     strategy:
+#       fail-fast: false
+#       matrix:
+#         language: [ 'csharp', 'javascript' ]
+#         # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
+#         # Learn more about CodeQL language support at https://git.io/codeql-language-support
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: csharp, javascript
+        # If you wish to specify custom queries, you can do so here or in a config file.
+        # By default, queries listed here will override any specified in a config file.
+        # Prefix the list here with "+" to use these queries and those in the config file.
+        # queries: ./path/to/local/query, your-org/your-repo/queries@main
+    - name: Build dd-trace-dotnet
+      run: |
+        ./tracer/build.sh
```

If only it were this simple 😂 
```suggestion
    - name: Build dd-trace-dotnet
      run: |
        ./tracer/build.sh BuildTracerHome BuildProfilerHome BuildNativeLoader ZipMonitoringHome
```